config/provider_config.go

```diff
@@ -123,33 +121,33 @@ func (c *ProviderConfig) Init(rc *RootConfig) error {
 		serviceConfig.adaptiveService = c.AdaptiveService
 	}
 
-	for k, v := range rc.Protocols {
-		if v.Name == tripleConstant.TRIPLE {
-			// Auto create grpc based health check service.
-			//healthService := NewServiceConfigBuilder().
-			//	SetProtocolIDs(k).
-			//	SetNotRegister(true).
-			//	SetInterface(constant.HealthCheckServiceInterface).
-			//	Build()
-			//if err := healthService.Init(rc); err != nil {
-			//	return err
-			//}
-			//c.Services[constant.HealthCheckServiceTypeName] = healthService
-
-			// Auto create reflection service configure only when provider with triple service is configured.
-			tripleReflectionService := NewServiceConfigBuilder().
-				SetProtocolIDs(k).
-				SetNotRegister(true).
-				SetInterface(constant.ReflectionServiceInterface).
-				Build()
-			if err := tripleReflectionService.Init(rc); err != nil {
-				return err
-			}
-			// Maybe only register once, If setting this service, break from traversing Protocols.
-			c.Services[constant.ReflectionServiceTypeName] = tripleReflectionService
-			break
-		}
-	}
+	//for k, v := range rc.Protocols {
```

Comment:
For this comment part, If users make use of ```config.Load```, reflection service would take effect?

Reply:
Yes, we still need to consider the old logic, but for now, our priority is to adapt to the new logic first.I will submit another pull request to adapt the old logic.

Reply:
Such as saving metadata in the old logic.